src/helpers/charts/bar_chart.ts

```diff
@@ -142,7 +147,7 @@ export class BarChart extends AbstractChart {
   }
 
   getDefinition(): BarChartDefinition {
-    return this.getDefinitionWithSpecificDataSets(this.dataSets, this.labelRange);
+    return this.getDefinitionWithSpecificDataSets(this.fullDataSets, this.labelRange);
```

Comment:
I don't understand the fix anymore :/
in a `DataSet`, `dataRange` includes `labelRange`...so it should end up being the same dataset string :thinking: 
I'll wait until you come back

Reply:
Some data is lost inside `createDataSets` when there's a single cell range and it includes (only) the title => the dataset is dropped (it should not)


https://github.com/odoo/o-spreadsheet/pull/2442